### DISCUSSION TITLE
All lowercase quotes, and "Instant Messenger" mode

### DISCUSSION THREAD

**Discussion by DrLeh** [monkeytypegame/monkeytype#5847] [Ideas] [answered]
Much of the typing that people do today is over instant messengers like Slack, Teams, etc. When people type in these systems they rarely will use casing as it's just a quick, informal message. When you see correct casing, especially at the start of words, it usually indicates that the person is typing from a phone that adds the casing automatically.

I'd like to be able to practice typing real coherent sentences (like quotes), but be able to ignore casing as that's how i'll end up typing in a real setting.

Thus I propose adding a new format that is like quotes, but all the letters have been lowercased. Punctuation can still be included as that is used for longer messages.

Another aspect of instant messaging is that often sentences will be sent one at a time, and thus would be terminated with `Enter` rather than `.`. This could be another new mode where there is no end-of-sentence puncutation, but instead each sentence from a quote is ended in a newline.

**Comment by ghost** [comment]
it got added !

**Comment by fehmer** [comment, marked as answer]
This was added with #5850 and will be available with the next release.

**Reply by DrLeh** [reply]
Wow that was quick!! Thanks!!
